test/unit/api_spec.js

```diff
@@ -2242,11 +2242,15 @@ page 1 / 3`);
       );
       const pdfDoc = await loadingTask.promise;
       const pdfPage = await pdfDoc.getPage(1);
+      // Wait for font data to be loaded so we can check that the font names
+      // match.
+      await pdfPage.getOperatorList();
```

Comment:
This is actually changing the test subtly, since the font loading now happens differently than before.
To avoid that, I believe it'd be more correct to place this (and the new `expect` check) just before the loadingTask-destruction instead.